tests/iscsi/iscsi_client.pm

```diff
@@ -29,13 +30,19 @@ sub run {
     type_string "yast2 iscsi-client\n";
     assert_screen 'iscsi-client', 180;
     mutex_lock('iscsi_ready');    # wait for server setup
+    wait_still_screen(2, 10);
+    unless (is_sle('<15') || is_leap('<15.1')) {
+        change_service_configuration(
+            after_writing => {start         => 'alt-f'},
```

Comment:
Hmm, service status is `partly-active`, I guess it's because it's activated by socket, so I would say it's fine to keep current state for client, we also don't really have to change on reboot setting, so to be honest, I would skip these steps here.